testdata/project-v4-with-plugins/.github/workflows/test-chart.yml

```diff
@@ -46,6 +46,12 @@ jobs:
         run: |
           helm lint ./dist/chart
 
+      - name: Install Prometheus Operator CRDs
+        run: |
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+          helm repo update
+          helm install prometheus-crds prometheus-community/prometheus-operator-crds
+
```

Comment:
@monteiro-renato 
this one need to be commented and under # TODO: Uncomment if Prometheus is enabled
We should only install it if the chart enabled the option

Reply:
I thought it would be a bit easier to just install the CRDs all the time as they shouldn't add too much time to the action run.

I've shifted them to be under the TODO now 👍.